test/custom-elements-es5-adapter.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable */
```

Comment:
This is the ES5  adapter needed to use ES5 "classes" with native custom elements.